TrustWalletCore.podspec

```diff
@@ -18,7 +18,7 @@ Pod::Spec.new do |s|
     submodules: true
   }
 
-  protobuf_dir = 'build/protobuf/staging/protobuf-3.7.0'
+  protobuf_dir = 'build/protobuf/staging/protobuf-3.7.1'
```

Comment:
for upgrading protobuf, can you do that in a separate PR?

Reply:
ok, I will revert it  to 3.7.0.